linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/rule.yml

```diff
@@ -5,8 +5,9 @@ prodtype: rhel7
 title: 'Enable FIPS Mode in GRUB2'
 
 description: |-
-    To ensure FIPS mode is enabled, rebuild <tt>initramfs</tt> by running the following command:
-    <pre>dracut -f</pre>
+    To ensure FIPS mode is enabled, install package <tt>dracut-fips</tt>, and rebuild <tt>initramfs</tt> by running the following commands:
+    <pre>yum install -y dracut-fips
```

Comment:
Although this rule currently only applies to `rhel7`, I think it makes sense to use `{{{ pkg_manager }}}` instead of `yum` command.